### ISSUE TITLE
ng breaks with `1.3.0-rc.4`

### ISSUE THREAD

**Issue by cexbrayat** [angular/angular-cli#7266]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
`1.3.0-rc.4`
`ng -v`breaks (see below).

`1.3.0-rc.3` works correctly and outputs:
```
@angular/cli: 1.3.0-rc.3
node: 7.4.0
os: darwin x64
```

1.2.7 does work too.

### Repro steps.
```
npm uninstall -g @angular/cli && npm i -g @angular/cli@1.3.0-rc.4
ng -v
```

### The log given by the failure.
`ng -v` breaks with:
```
ng -v                                                                                                    ✹ ✭
module.js:472
    throw err;
    ^

Error: Cannot find module '../models/config'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/ced-pro/.nvm/versions/node/v7.4.0/lib/node_modules/@angular/cli/bin/ng:7:19)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

Same for `ng new`.

**Comment by jinder** [comment]
Also seeing this in rc4 for 'ng serve'.

**Comment by hansl** [comment]
Thanks. Looking.

**Comment by hansl** [comment]
This has been fixed in Angular CLI 1.3.0-rc.5.

**Comment by cexbrayat** [comment]
I can confirm. Thank you @hansl !
